features/121-Portal/Change_password.feature

```diff
@@ -0,0 +1,34 @@
+@ho-portal
+
+Feature: Change password
+
+  Background:
+    Given a logged-in user
+    And "Change password" page is open
+
+  Scenario: Change password succesfully
+    Given user has entered current password correctly
+    Given Existing password field apperas green when active
+    When user enters new password
+    And new password is at least 8 characters one capital letter, one lowercase letter and one number
```

Comment:
I think the rule is here you cannot have 'And's after a When. It should be immediately followed by a Then. So I think you should transform most of these to Given's and then take as When the action of clicking 'Update'.